ui/raidboss/data/05-shb/raid/e6n.js

```diff
@@ -6,8 +6,271 @@
     ko: /^희망의 낙원 에덴: 공명편 \(2\)$/,
   },
   timelineFile: 'e6n.txt',
+  timelineTriggers: [
+    {
+      // We warn the user here because the startsUsing warning gives only 3.5s or so.
+      id: 'E6N Downburst',
+      regex: /Downburst/,
+      beforeSeconds: 5,
+      response: Responses.knockback('info'),
+    },
+
+  ],
   triggers: [
+    {
+      id: 'E6N Superstorm',
+      regex: Regexes.startsUsing({ source: 'Garuda', id: '4BD7', capture: false }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E6N Ferostorm',
+      regex: Regexes.startsUsing({ source: ['Garuda', 'Raktapaska'], id: ['4BD[DEF]', '4BE[345]'], capture: false }),
+      infoText: {
+        en: 'Avoid green nails',
```

Comment:
Do you know which ways each of these abilities go? I wanted to call left/right on these, both here and in savage...

Reply:
Unfortunately I don't. I'll check for that as I have time, but I don't record, so it's difficult without an organized group.

Reply:
Ok! You can also always upload to fflogs and look at the playback and see where people move.

Reply:
I'll try that tomorrow, thanks for the suggestion.